apps/docs/src/.vitepress/components/ColorPaletteValue.vue

```diff
@@ -0,0 +1,126 @@
+<script lang="ts" setup>
+import CopyIcon from "./icons/CopyIcon.vue";
+
+export type ColorPaletteValueProps = {
+  /** Text to show below the value. */
+  description: number | string;
+  /** CSS variable of the color value to use as background color. */
+  color: string;
+  /** Text color (should have good contrast to `color` property). */
+  textColor: string;
+  /** Optional name to show inside of the color. */
+  name?: string;
+  /** If true, a border is shown around the color value. Useful for very light/dark colors with low contrast. */
+  showBorder?: boolean;
+};
+
+const props = defineProps<ColorPaletteValueProps>();
+
+const emit = defineEmits<{
+  select: [];
+}>();
+</script>
+
+<template>
+  <button class="step" @keyup.enter="emit('select')" @click="emit('select')">
```

Comment:
Please remove redundant event handle: Pressing enter on a focused button triggers a click.
```suggestion
  <button class="step" @click="emit('select')">
```

Reply:
Nice didn't know that!